Changes

```diff
@@ -497,6 +497,9 @@ Working version
   -runtime-search used for bytecode executables produced by the compiler.
   (David Allsopp, review by Damien Doligez and Samuel Hym)
 
+- #13705: Cache test results of custom Autoconf tests from aclocal.m4.
+  (Antonin Décimo, review by David Allsopp)
```

Comment:
This should be a bit further up the section, which I mention because you could add Miod's name as you move it :)